website/docs/intro/migration/terraform-1.6.mdx

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 5
+sidebar_label: Migrating from Terraform 1.6.x
+description: |-
+    Learn how to migrate to OpenTofu from Terraform.
+---
+
+# Migrating to OpenTofu from Terraform 1.6.x
+
+When migrating from Terraform 1.6.x, please migrate to OpenTofu 1.6.2 first, then
+[upgrade your OpenTofu installation](../upgrading.mdx) to the latest version.
+
+OpenTofu 1.6.2 is largely compatible with Terraform 1.6.x with a few minor changes needed. This migration guide will
+take you through the process of switching from Terraform to OpenTofu.
+
+## Step 0: Prepare a disaster recovery plan
+
+Although OpenTofu 1.6.2 is very similar to Terraform 1.6.6, it is a non-trivial change affecting your infrastructure.
```

Comment:
I'm not sure about the "non-trivial" bit, especially since all over the place we're calling it a easy drop-in replacement.

WDYT about just 
> Although OpenTofu 1.6.2 is very similar to Terraform 1.6.6, make sure you have an up to date and *tested* disaster recovery plan.

But not a strong opinion, fine with keeping it this way too.